tests/app/Utils/feverUtilTest

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+use PHPUnit\Framework\TestCase;
+
+class feverUtilTest extends TestCase
+{
+	public function testDeleteKey(): void
+	{
+		$bool = FreshRSS_fever_Util::checkFeverPath();
+		$this->assertIsBool($bool);
+		$this->assertTrue($bool);
+	}
+
+	public function testKeyPathExpectContextException(): void
+	{
+		$this->expectException(FreshRSS_Context_Exception::class);
+		FreshRSS_fever_Util::getKeyPath('foo');
+	}
+
+	public function testUpdateKeyExpectContextException(): void
+	{
+		$this->expectException(FreshRSS_Context_Exception::class);
+		$this->expectExceptionMessage('System configuration not initialised!');
+		FreshRSS_fever_Util::updateKey('foo', 'bar');
```

Comment:
To be more precise, this is an example of test, which I find more negative than positive.
It relies on some internal behaviour, which happens to be true at the moment, but which are not meant to be stable.
There is no value (or worse, a negative value), in testing the precise exception and exception message for internal behaviour.

For examples of positive unit tests in my book, see e.g. https://github.com/FreshRSS/FreshRSS/blob/edge/tests/app/Models/SearchTest.php , https://github.com/FreshRSS/FreshRSS/blob/edge/tests/app/Utils/passwordUtilTest.php

Reply:
Let me rephrase: adding those tests do not contribute to higher quality code, but on the contrary, would make the code base more energy-demanding to maintain.
I very much welcome more tests. Just not this type, sorry...